etc/chaos_supervisor.conf

```diff
@@ -1,5 +1,5 @@
 [program:chaos]
-command=/root/.virtualenvs/chaos/bin/python chaos.py
+command=bash chaos_wrapper.sh
```

Comment:
`./chaos_wrapper.sh` should also work, provided that `chaos_wrapper.sh` has the executable bit set, since `#!/bin/bash` is at the top of `chaos_wrapper.sh`.